### PR TITLE
add gibberish-aes@1.0.0 w/ npm auto-update

### PR DESCRIPTION
**Pull request by iwdmb**
#8741
#8785

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required): https://github.com/mdp/gibberish-aes
- GitHub:
  - Count of stars: 416
  - Count of watchers: 35
  - Count of forks: 93
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [x] The lib has notable popularity
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/gibberish-aes/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "filename": "gibberish-aes.min.js",
+  "version": "1.0.0",
+  "author": "Mark Percival <mark@mpercival.com>",
+  "description": "A lightweight Javascript Libray for OpenSSL compatible AES CBC encryption.",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/mdp/gibberish-aes.git"
+  },
+  "license": "MIT",
+  "npmName": "gibberish-aes",
```
Comment (pvnr0082t) [inline ajax/libs/gibberish-aes/package.json]: @iwdmb https://www.npmjs.com/package/gibberish-aes seems like not officially maintained. Could you please confirm this? If it's not officially maintained, we should use git auto-update config.

File: ajax/libs/gibberish-aes/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "gibberish-aes",
+  "filename": "gibberish-aes.min.js",
```
Comment (x09326) [inline ajax/libs/gibberish-aes/package.json]: @iwdmb 
I saw the filename in the `dist` is `gibberish-aes-1.0.0.min.js`.
If we want to use auto-update, the filename couldn't with version number.
Would you ask the author to change the main filename without version number? Thanks.

Comment (PeterDaveHello) [inline ajax/libs/gibberish-aes/package.json]: @x09326 you should use `comment` or `request change` here if it's not good enough to merge immediately

Comment (x09326) [inline ajax/libs/gibberish-aes/package.json]: Sorry, I will remember that next time.

Comment (PeterDaveHello) [inline ajax/libs/gibberish-aes/package.json]: @x09326 not too late to request for changes now :D

**Pull request closed without merging**